python/feathub/feature_views/derived_feature_view.py

```diff
@@ -98,7 +104,8 @@ def build(
                 table_desc = registry.get_features(name=join_table_name)
                 join_feature = table_desc.get_feature(feature_name=join_feature_name)
                 if source.name == join_table_name:
-                    feature = join_feature
+                    feature = deepcopy(join_feature)
```

Comment:
Can you explain why we should copy the `input_features` from the `join_features`?

Reply:
Is this comment addressed?

Reply:
My bad. I misunderstood the comment and addressed the wrong problem. I think it is resolved now.